x-pack/solutions/security/plugins/security_solution/scripts/endpoint/common/response_actions.ts

```diff
@@ -347,21 +355,23 @@ export async function getLatestActionDoc(
   esClient: Client
 ): Promise<SearchHit<EndpointAction> | undefined> {
   return (
-    await esClient.search<EndpointAction>({
-      index: AGENT_ACTIONS_INDEX,
-      ignore_unavailable: true,
-      query: {
-        match: {
-          type: 'INPUT_ACTION',
+    await esClient
+      .search<EndpointAction>({
+        index: AGENT_ACTIONS_INDEX,
+        ignore_unavailable: true,
+        query: {
+          match: {
+            type: 'INPUT_ACTION',
+          },
         },
-      },
-      sort: {
-        '@timestamp': {
-          order: 'desc',
+        sort: {
```

Comment:
just to confirm, since the change in github seems strange. sort is still at the same level as query right?

Reply:
Good! GH showed me that the line was removed 🤷  But my assumption was that nothing changed but the whitespace, thanks! 👍